README.md

```diff
@@ -178,6 +178,12 @@ iban.ToString("F");// NL20 INGB 0001 2345 67
 Represents the amount and the currency. Technically this is not SVO. However it
 acts identically as a SVO.
 
+``` C#
+Money money = 125.34 + Currency.EUR;
+var sum = (12 + Currency.EUR) + (15 + Currency.EUR); // Throws CurrencyMismatchException()
```

Comment:
throws an exception, but the currencies are the same?

Reply:
Oops.